contracts/HubPool.sol

```diff
@@ -149,9 +149,9 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     event ProposeRootBundle(
         uint32 challengePeriodEndTimestamp,
         uint64 poolRebalanceLeafCount,
-        uint256[] bundleEvaluationBlockNumbers,
+        uint256[] indexed bundleEvaluationBlockNumbers,
```

Comment:
I don't understand why we would need to index this param? We would only need to index by this if we _knew_ the bundle evaluation block numbers and wanted to find the event based on that?

Reply:
> We would only need to index by this if we knew the bundle evaluation block numbers and wanted to find the event based on that?

Yeah well I was thinking this would be a real situation, we can just ignore it for now.

Reply:
> We would only need to index by this if we knew the bundle evaluation block numbers and wanted to find the event based on that?

Yeah well I was thinking this would be a real situation, we can just ignore it for now.